analyzer/CMakeLists.txt

```diff
@@ -119,7 +119,7 @@ include_directories(${AR_INCLUDE_DIR})
 find_package(FrontendLLVM REQUIRED)
 include_directories(${FRONTEND_LLVM_INCLUDE_DIR})
 
-find_package(LLVM 14.0.0...<15.0.0 REQUIRED HINTS '/usr/lib/llvm14/lib/cmake/llvm')
+find_package(LLVM 14.0.0...<15.0.0 REQUIRED)
```

Comment:
Unfortunately, it seems like version ranges in `find_package` are only available since CMake 3.19:
https://cmake.org/cmake/help/latest/command/find_package.html#basic-signature

We currently support cmake 3.4.3 and above, as specified by `cmake_minimum_required`:
https://github.com/NASA-SW-VnV/ikos/blob/master/analyzer/CMakeLists.txt#L41

We could potentially increase that, although there might be people depending on lower versions.
We could also have two versions of the `find_package` call depending on the cmake version.

Reply:
Removed version ranges in [5ab5b3a](https://github.com/NASA-SW-VnV/ikos/pull/308/commits/5ab5b3aa3316782e44338b617ee6edea253dc11e).

It looks like they were redundant, as `FindLLVM.cmake` returns the first `LLVM` version it finds, and it always begins it's search by looking for `llvm-config-14`.